btcpay-server/docker-compose.yml

```diff
@@ -8,66 +8,69 @@ services:
 
   web:
-    image: btcpayserver/btcpayserver:1.5.4@sha256:2c1338e14cda481a738f900f1684167d435618db3890bcfed6e51cd7e8d7674c
+    image: btcpayserver/btcpayserver:1.6.3@sha256:fb514c417933fc088dec2fb841f5d3572e35429412e79bb12b25fb4437519162
     user: "1000:1000"
     restart: on-failure
     stop_grace_period: 1m
-    depends_on: [ nbxplorer, postgres ]
-    entrypoint: [ "dotnet", "BTCPayServer.dll" ]
+    depends_on: [nbxplorer, postgres]
+    entrypoint: ["dotnet", "BTCPayServer.dll"]
     volumes:
-        - ${APP_DATA_DIR}/data/btcpay:/data
-        - ${APP_DATA_DIR}/data/nbxplorer:/data/.nbxplorer
-        - ${APP_LIGHTNING_NODE_DATA_DIR}:/lnd:ro
+      - ${APP_DATA_DIR}/data/btcpay:/data
+      - ${APP_DATA_DIR}/data/nbxplorer:/data/.nbxplorer
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/lnd:ro
     environment:
-        HOME: "/data"
-        BTCPAY_DATADIR: "/data"
-        BTCPAY_PLUGINDIR: "/data/plugins"
-        BTCPAY_DOCKERDEPLOYMENT: "false"
-        BTCPAY_POSTGRES: "User ID=postgres;Host=$APP_BTCPAY_SERVER_DB_IP;Port=5432;Database=btcpayserver$APP_BITCOIN_NETWORK"
-        BTCPAY_NETWORK: "$APP_BITCOIN_NETWORK"
-        BTCPAY_BIND: 0.0.0.0:$APP_BTCPAY_SERVER_PORT
-        BTCPAY_CHAINS: "btc"
-        BTCPAY_BTCEXPLORERURL: "http://$APP_BTCPAY_SERVER_NBXPLORER_IP:32838"
-        BTCPAY_BTCLIGHTNING: "type=lnd-rest;server=https://$APP_LIGHTNING_NODE_IP:$APP_LIGHTNING_NODE_REST_PORT/;macaroonfilepath=/lnd/data/chain/bitcoin/$APP_BITCOIN_NETWORK/admin.macaroon;allowinsecure=true"
-        BTCPAY_SOCKSENDPOINT: $TOR_PROXY_IP:$TOR_PROXY_PORT
+      HOME: "/data"
+      BTCPAY_DATADIR: "/data"
+      BTCPAY_PLUGINDIR: "/data/plugins"
+      BTCPAY_DOCKERDEPLOYMENT: "false"
+      BTCPAY_POSTGRES: "User ID=postgres;Host=$APP_BTCPAY_SERVER_DB_IP;Port=5432;Database=btcpayserver$APP_BITCOIN_NETWORK"
+      BTCPAY_NETWORK: "$APP_BITCOIN_NETWORK"
+      BTCPAY_BIND: 0.0.0.0:$APP_BTCPAY_SERVER_PORT
+      BTCPAY_CHAINS: "btc"
+      BTCPAY_BTCEXPLORERURL: "http://$APP_BTCPAY_SERVER_NBXPLORER_IP:32838"
+      BTCPAY_BTCLIGHTNING: "type=lnd-rest;server=https://$APP_LIGHTNING_NODE_IP:$APP_LIGHTNING_NODE_REST_PORT/;macaroonfilepath=/lnd/data/chain/bitcoin/$APP_BITCOIN_NETWORK/admin.macaroon;allowinsecure=true"
+      BTCPAY_SOCKSENDPOINT: $TOR_PROXY_IP:$TOR_PROXY_PORT
     networks:
       default:
         ipv4_address: $APP_BTCPAY_SERVER_IP
 
   postgres:
-    image: btcpayserver/postgres:13.6@sha256:dadf0048895a888d88a2dd773dde2f5868c45f74ad37c6d208694df54b590531
+    image: btcpayserver/postgres:13.7@sha256:dadf0048895a888d88a2dd773dde2f5868c45f74ad37c6d208694df54b590531
     # This needs to run as root for migrations to succeed
     # user: "1000:1000"
     restart: on-failure
     # https://github.com/btcpayserver/btcpayserver-docker/commit/a65e7db6851092c75c5ac6c091a5f36ccc5fc26e
-    command: [ "-c", "random_page_cost=1.0" ]
+    command: ["-c", "random_page_cost=1.0", "-c", "shared_preload_libraries=pg_stat_statements"]
```

Comment:
Out of interest, how come this `"shared_preload_libraries=pg_stat_statements"` was added? Is BTCPay now reporting DB stats?

Reply:
I've taken this change from here https://github.com/btcpayserver/btcpayserver-docker/commit/a073a5f7b3a010165e882185d6f39c581552c11c. The Dockerfile and entrypoint are available [here](https://github.com/btcpayserver/dockerfile-deps/tree/master/Postgres/13.7). 

The only [results](https://github.com/search?q=org%3Abtcpayserver+pg_stat_statements&type=issues) searching for `pg_stat_statements` in the `btcpayserver` org is https://github.com/btcpayserver/btcpayserver/pull/3688. I guess it was added for debugging purposes.